source/examples.rst

```diff
@@ -9,4 +9,12 @@ Text to image GUI
 
 See : `EMF Demos - Text to Image GUI <https://github.com/easy-model-fusion/demos/tree/main/text-to-image-gui>`_
 
-The code defines a graphical user interface (GUI) application using the Tkinter library. This application allows users to generate images from text using the `Stabilityai - Stable Diffusion XL Base 1.0 <https://huggingface.co/stabilityai/stable-diffusion-xl-base-1.0>`_ model.
\ No newline at end of file
+The code defines a graphical user interface (GUI) application using the Tkinter library. This application allows users to generate images from text using the `Stabilityai - Stable Diffusion XL Base 1.0 <https://huggingface.co/stabilityai/stable-diffusion-xl-base-1.0>`_ model.
+
+
+Text model to image model GUI
```

Comment:
not quite sure the title is good enough
makes me think that we are migrating from one model to another through a GUI which doesnt make much sense
however i assume it is a text generation model generates a text which is then used as an input for a text to image model ?